src/cascade/input_data/configuration/form.py

```diff
@@ -63,11 +73,11 @@ def _full_form_validation(self, root):  # noqa: C901 too complex
                 eta = self.eta
 
             if self.density == "uniform":
-                self.prior_object = priors.Uniform(lower, upper, mean)
+                self.prior_object = priors.Uniform(lower, upper, mean, eta=eta)
```

Comment:
I don't think eta affects uniform, does it? eta affects the prior for log-Gaussian, log-Laplace, and log-Students.

Reply:
I don't think there is a log-uniform. So I don't think this one gets the eta.

Reply:
Brad's docs seem to indicate that it effects any prior, but I don't really follow what he's saying:

https://bradbell.github.io/dismod_at/doc/prior_table.htm

Reply:
There are two parts to the section about eta. The first says it applies only to log-transformed values. The second is confusing, but it looks to me like an offset for those same log-transformed values. So I think it's not on the Uniform distribution.

Reply:
Ok. I'll pull them from the inappropriate distributions.